Create TurnDistance.java

Triston-Van-Wyk opened the pull request:
This is a fix to issue #19

Commit:
Create TurnDistance.java

devinvs commented on src/main/java/frc/robot/commands/TurnDistance.java:
```diff
@@ -0,0 +1,61 @@
+  private int targetDistance;
+  private double currentPosition;
+
+  private int allowedErroe = 100;
```
Typeo

devinvs commented on src/main/java/frc/robot/commands/TurnDistance.java:
```diff
@@ -0,0 +1,61 @@
+  @Override
+  public void initialize() {
+    drivetrain.setPositionZero();
+    drivetrain.set(ControlMode.MotionMagic, targetDistance, targetDistance);
```
I believe this needs to be in execute... Deploy it to the robot and test it before you change it.

devinvs commented on src/main/java/frc/robot/commands/TurnDistance.java:
```diff
@@ -0,0 +1,61 @@
+      return true;
+    } else{
+      return false;
+    }
```
The first if statement is completely unnecessary, we don't care about that. Also, you don't want to use `==` because that will only work when the error is exactly the allowed error. We want it to act as a range, so use `<` or `>`

Commit:
Update TurnDistance.java

I didn't test  the drivetrain.stePositionZero(); and drivetrain.set(ContorlMode.MotionMagic, targetDistance, targetDistance); code on the robot. So we will have to still do that.

Triston-Van-Wyk commented:
I updated the command. We will still need to test the code I moved from initialize to exicute.
